examples/this-that/wasm/src/lib.rs

```diff
@@ -0,0 +1,30 @@
+#![no_std]
+
+use gstd::msg;
+use parity_scale_codec::Encode;
+use this_that_app::{commands, queries};
+
+// TODO: This function might be generated by some macro like `gprogram`
+#[no_mangle]
+extern "C" fn handle() {
+    let command = msg::load::<commands::Commands>()
+        .expect("This needs to be handled in a consistent way: input parse error");
+    let (result, is_error) = commands::handlers::handle_commands(command);
+    let result = result.encode();
+    if is_error {
+        unsafe {
+            gsys::gr_panic(result.as_ptr(), result.len() as u32);
+        }
+    }
+    msg::reply(result, 0).expect("This needs to be handled in a consistent way: reply error");
```

Comment:
why 0 gas ?

Reply:
This is not gas, but value. As far as I get transferring value from a contract is not used